projects/developer/src/app/common/components/recipe-graph/component.ts

```diff
@@ -183,6 +190,35 @@ export class RecipeGraphComponent implements OnInit, OnChanges, AfterViewInit {
         this.update.next(true);
     }
 
+    private updateText() {
+        const initialTheme = this.themeService.getActiveTheme().name;
+        let initialTextColor = ColorService.FONT_LIGHT_THEME; // default
+        switch (initialTheme) {
+            case 'dark':
+                initialTextColor = ColorService.FONT_DARK_THEME;
+                break;
+        }
+        this.chartOptions.title.fontColor = initialTextColor;
+        this.chartOptions.scales.yAxes[0].ticks.fontColor = initialTextColor;
+        this.chartOptions.scales.xAxes[0].ticks.fontColor = initialTextColor;
+
+        this.themeSubscription = this.themeService.themeChange.subscribe(theme => {
+                let textColor = ColorService.FONT_LIGHT_THEME; // default
+                switch (theme.name) {
+                    case 'dark':
+                        textColor = ColorService.FONT_DARK_THEME;
+                        break;
+                }
+                this.chartOptions.title.fontColor = textColor;
+                this.chartOptions.scales.yAxes[0].ticks.fontColor = textColor;
+                this.chartOptions.scales.xAxes[0].ticks.fontColor = textColor;
+                setTimeout(() => {
+                    this.chartNodeJobs.reinit();

```

Comment:
Did you try using just `.update({duration: 0})` instead of `.reinit()`?

Reply:
UIChart doesn't appear to have an `update()` method, it does have a `refresh()` method but I'd didn't have any luck with it.